attestapis/buf.md

```diff
@@ -0,0 +1,38 @@
+# AttestAPIs
+
+The APIs required to interact with the `AttestService`.
```

Comment:
I'm up for suggestions on these docs.

Also you can see the rendering [here](https://github.com/mobilecoinfoundation/probufs/blob/nick/attest-proto/attestapis/buf.md)